packages/atomic/src/components/commerce/product-template-components/atomic-product-image/atomic-product-image.tsx

```diff
@@ -157,6 +157,27 @@ export class AtomicProductImage implements InitializableComponent<Bindings> {
     }) as Image[];
   }
 
+  private isImageAltFieldValid() {
+    if (typeof this.imageAlt !== 'string') {
+      return false;
+    }
+
+    if (!this.imageAltField) {
+      return false;
+    }
+
+    const value = ProductTemplatesHelpers.getProductProperty(
+      this.product,
+      this.imageAltField
+    );
+
+    if (isNullOrUndefined(value)) {
+      return false;
+    }
+
+    return true;
```

Comment:
```suggestion
    return !!this.imageAlt
```

I think this would be equivalent, because `this.imageAlt` returns a `string` or a `string[]` when valid, and `null` (so falsy) otherwise.